packages/editor/src/plugins/image/index.ts

```diff
@@ -23,6 +23,8 @@ const imageState = object({
   src: upload(''),
   link: optional(object({ href: string('') })),
   alt: optional(string('')),
+  imageSource: optional(string('')),
+  licence: optional(string('')),
```

Comment:
Hmmm, as I said yesterday in out call, it feels weird to store a integer based id in a string here…

Please use strings as keys in your array or an enum or an object (`ImageLicense.Pixabay` would be a nice, readable way to call it) or if you want to keep the numeric ids please store them as a number as well…

Reply:
Done, thanks :+1:

Reply:
Nice!